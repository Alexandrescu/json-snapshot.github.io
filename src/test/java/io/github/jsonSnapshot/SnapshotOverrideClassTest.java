package io.github.jsonSnapshot;

import org.junit.AfterClass;
import org.junit.BeforeClass;

import static io.github.jsonSnapshot.SnapshotMatcher.start;
import static io.github.jsonSnapshot.SnapshotMatcher.validateSnapshots;

public class SnapshotOverrideClassTest extends SnapshotSuperClassTest {

    @BeforeClass
    public static void beforeAll() {
        start();
    }

    @AfterClass
    public static void afterAll() {
        validateSnapshots();
    }

    @Override
    public String getName() {
        return "anyName";
    }
}
