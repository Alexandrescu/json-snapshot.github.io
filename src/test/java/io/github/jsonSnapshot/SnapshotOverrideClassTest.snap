io.github.jsonSnapshot.SnapshotOverrideClassTest.shouldMatchSnapshotOne=[
  "anyName"
]